src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -194,7 +195,7 @@ void HolonomicDrive::Run()
             SlowMode();
         }
 
-        if (abs(rotate) > 0.05)
+        if (m_previousDriveState == (ChassisOptionEnums::DriveStateType::DRIVE_TO_CORAL_STATION || ChassisOptionEnums::DriveStateType::DRIVE_TO_LEFT_REEF_BRANCH || ChassisOptionEnums::DriveStateType::DRIVE_TO_RIGHT_REEF_BRANCH))
```

Comment:
You need to keep the old logic there, you just need to add this logic as a OR condition. Also you never set the m_previousDriveState